ShiftLogger.Mefdev/ShiftLoggerUI/Inputs/UserInput.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Globalization;
+using Spectre.Console;
+
+namespace ShiftLogger.Mefdev.ShiftLoggerUI.Inputs;
+
+public class UserInput
+{
+    public UserInput()
```

Comment:
🟠 **Redundant Code**

💡 Is a blank constructor required? If not, remove. If so, perhaps leave a comment inside the constructor explaining why.